### PR TITLE
Actually fix all the configmap nonsense

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/assign @petr-muller 

I moved the periodics so we could continue to honor the assumption that a job for $repo/$org is in the right dir

### CODE REVIEW

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ periodics:
         valueFrom:
           configMapKeyRef:
             key: openshift-openshift-ansible-release-3.11.yaml
-            name: ci-operator-3.x-configs
+            name: ci-operator-misc-configs
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml]: And these are periodics that reference 3.11 configs of `openshift/openshift-ansible`...

```suggestion
            name: ci-operator-3.x-configs
```

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml [hunk 2]
```diff
@@ -70,7 +70,7 @@ periodics:
...
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml]:
```suggestion
            name: ci-operator-3.x-configs
```

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml [hunk 3]
```diff
@@ -124,7 +124,7 @@ periodics:
...
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml]:
```suggestion
            name: ci-operator-3.x-configs
```

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml [hunk 4]
```diff
@@ -171,7 +171,7 @@ periodics:
...
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml]:
```suggestion
            name: ci-operator-3.x-configs
```

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml (hunk 1 above)
Comment (stevekuznetsov) [reply ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml]: for fuck's sake

**Pull request closed without merging**